packages/cli/src/commands/init.ts

```
@@ -1,29 +1,50 @@
 import { Command } from "commander";
 import { spawn } from "cross-spawn";
+import fs from "fs";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)